package Test0309Practice;

import java.io.IOException;
import java.net.ServerSocket;
import java.net.Socket;

public class MainTest {
	private static Socket s;
	private static ServerSocket ss=null;
	public static void main(String[] args) {
		while(true){
			try {
				ss=new ServerSocket(5500);
				s=ss.accept();
				System.out.println("已成功连接");
			} catch (IOException e) {
				// TODO Auto-generated catch block
				e.printStackTrace();
			}
		}	
	}
}
